src-built-in/components/toolbar/src/app.jsx

```diff
@@ -12,20 +12,27 @@ import React from "react";
 import { ToolbarShell } from "@chartiq/finsemble-ui/lib/components/toolbar/ToolbarShell";
 import { onReady } from "@chartiq/finsemble-ui/lib/effects/onReadyEffects";
 import { DragHandle } from '@chartiq/finsemble-ui/lib/components/toolbar/DragHandle'
-import { DummyExport } from '@chartiq/finsemble-ui/lib/components/dummyExport'
+import { RevealAll } from '@chartiq/finsemble-ui/lib/components/toolbar/revealAll';
 import '../toolbar.css';
 const Toolbar = () => {
 	return (
 		<ToolbarShell>
-			<DragHandle/>
-			<DummyExport/>
-			{/* Drag Handle */}
-			{/* Workspace Management Menu */}
-			{/* App Menu */}
-			{/* Favorites Section */}
-			{/* Minimize All */}
-			{/* Reveal All */}
-			{/* Auto Arrange */}
+			<div className="finsemble-toolbar-section left">
+				<DragHandle />
+				{/* Workspace Management Menu */}
+				{/* App Menu */}
+			</div>
+			<div className="finsemble-toolbar-section center">
+				{/* Favorites Section */}
+			</div>
+			<div className="finsemble-toolbar-section right">
+				<RevealAll />
+				{/* Minimize All */}
+				{/* Auto Arrange */}
```

Comment:
If we're disabling those two functionalities by commenting this code out, should we import them at the top of this file so people can just use them when those lines are uncommented?